test/unit/model/bulk-create.test.js

```diff
@@ -41,6 +47,16 @@ describe(Support.getTestDialectTeaser('Model'), () => {
           { account_id: 42, id: null },
         ]);
       });
+
+      it('skips setters if the raw option is true', async function () {
```

Comment:
Should we add a unit test that checks that the name is changed to `A` when raw is not there/false?

Reply:
Good catch, will add it